src/Feature.spec.tsx

```diff
@@ -107,14 +107,14 @@ describe('<OptimizelyFeature>', () => {
 
       // while it's waiting for onReady()
       expect(component.text()).toBe('')
-      resolver.resolve({ sucess: true })
+      resolver.resolve({ success: true })
```

Comment:
Success was misspelled in this entire test file, so all the tests were actually running as though success was `false` 🤦‍♂

Reply:
😱